docs/guides/CustomStartupGuide.md

```diff
@@ -96,6 +104,21 @@ Install the media utility tool:
    ```bash
    git clone https://github.com/edgexfoundry/device-usb-camera.git
    ```
+
+## OPTIONAL: Configuration Options
+### Configurable RTSP server hostname and port
+The hostname and port of the RTSP server can be configured in the `[Driver]` section of the `/cmd/res/configuration.toml` file. The default values can be used for this guide.
+
+For example:
+```yaml
+[Driver]
+  RtspServerHostName = "localhost"
+  RtspTcpPort = "8554"
+```
+<p align="left">
+      <i>Sample: Snippet from configuration.toml</i>
+</p>
```

Comment:
Is this different than just using:

```
_Sample: Snippet from configuration.toml_
```

_Sample: Snippet from configuration.toml_

Reply:
I'm not sure, I just know that this is how its done in the onvif docs. I believe Kimberly might have been the first to format it like that, but I'm not entirely sure.